tests/SprykerEcoTest/Zed/Unzer/Business/UnzerFacade/CreateUnzerCredentialsTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Unzer\Business\UnzerFacade;
+
+use Generated\Shared\DataBuilder\UnzerCredentialsBuilder;
+use SprykerEcoTest\Zed\Unzer\Business\UnzerFacadeBaseTest;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Unzer
+ * @group Business
+ * @group UnzerFacade
+ */
+class CreateUnzerCredentialsTest extends UnzerFacadeBaseTest
+{
+    /**
+     * @return void
+     */
+    public function testCreateUnzerCredentials(): void
```

Comment:
there is a possibility to get non-successful response
we need to cover it as well

Reply:
Fixed.